sdk/docs/manually-written/sdk/component-howtos/smart-contracts/flags.rst

```diff
@@ -1,112 +1,146 @@
-- | ``--multi-package-path FILE``
-  | Path to the multi-package.yaml file
+- | ``--package-root ARG``
 
-``test``
-=============
+  | :subsiteref:`DPM-specific<dpm>` option.
 
-- | ``--files FILE``
-  | Only run test declarations in the specified files.                     
-- | ``--all``
-  | Run tests in current project as well as dependencies
-- | ``--load-coverage-only``
-  | Don't run any tests - only load coverage results from files and write the aggregate to a single file.
-- | ``--show-coverage``
-  | Show detailed test coverage
-- | ``--color``
-  | Colored test results
-- | ``--junit FILENAME``
-  | Filename of JUnit output file
-- | ``-p,--test-pattern PATTERN``
-  | Only scripts with names containing the given pattern will be executed.
-- | ``--transactions-output ARG``
-  | Filename to which the transaction list should be output
-- | ``--load-coverage ARG``
-  | File to read prior coverage results from. Can be specified more than once.
-- | ``--save-coverage ARG``
-  | File to write final aggregated coverage results to.
-- | ``--coverage-ignore-choice ARG``
-  | Remove choices matching a regex from the coverage
-  | report. The full name of a local choice takes the
-  | format '<module>:<template name>:<choice name>',
-  | preceded by '<package id>:' for nonlocal packages.
+  | Path to the root of a :ref:`glos-daml-package` containing ``daml.yaml``.
+    Using this option, you set the package to build/test. You should prefer the
+    ``DAML_PACKAGE`` environment variable over this option. See
+    :subsiteref:`dpm-configuration` for more details.
+
+- | ``--project-root ARG``
+
+  | :subsiteref:`DPM-specific<dpm>` option.
+
+  | ``project-root`` is **deprecated**, please use ``--package-root``.
+
+  | Path to the root of a :ref:`glos-daml-package` containing ``daml.yaml``. You
+    should prefer the ``DAML_PACKAGE`` environment variable over this option.
+    See :subsiteref:`dpm-configuration` for more details.
+
+- | ``--package-check``
+
+  | Check if running in Daml :ref:`glos-daml-package`.
 
-Includes :ref:`General Daml Compiler flags <general-damlc-flags>`
+  | ?????
 
-General Daml Compiler flags
-===========================
-.. _general-damlc-flags:
+- | ``--project-check``
 
-The following flags are shared by many Daml commands, which reference this section above.
+  | ``project-check`` is **deprecated**, please use ``--package-check``.
+
+  | Check if running in Daml :ref:`glos-daml-package`. (project-check is
+    deprecated, please use `--package-check`).
 
 - | ``--include INCLUDE-PATH``
-  | Path to an additional source directory to be included
+
+  | Path to an additional source directory to be included.
+
 - | ``--package-db LOC-OF-PACKAGE-DB``
-  | Use package database in the given location
+
+  | Use :ref:`glos-package-database` in the given location.
+
 - | ``--access-token-file PATH``
+
   | Path to the token-file for ledger authorization.
+
+  | ?????
```

Comment:
I think this is needed when IDE is accessing the real ledger. It would be good to clarify that.
@samuel-williams-da can you help?